addons/sale/models/sale_order.py

```diff
@@ -1898,3 +1898,11 @@ def _is_paid(self):
         """
         self.ensure_one()
         return self.currency_id.compare_amounts(self.amount_paid, self.amount_total) >= 0
+
+    def _message_get_suggested_recipients(self):
+        recipients = super()._message_get_suggested_recipients()
+        for order in self.filtered('partner_id'):
+            order._message_add_suggested_recipient(
+                recipients, partner=order.partner_id, reason=_("Customer")
+            )
+        return recipients
```

Comment:
Wrong place for the method, it's not a TOOLING, should be in the `# MAIL #` section ;)

Reply:
You're right, I didn't notice those sections. Updated.